samples/subsys/shell/shell_module/prj.conf

```diff
@@ -8,3 +8,4 @@ CONFIG_BOOT_BANNER=n
 CONFIG_THREAD_NAME=y
 CONFIG_DEVICE_SHELL=y
 CONFIG_GPIO_SHELL=y
+CONFIG_HWINFO_SHELL=y
```

Comment:
I think you would want CONFIG_HWINFO=y here as well. Maybe put this in tests/boards/board_shell/, it more appropriate than the shell sample.

Reply:
I thought the shell sample should have all shell command built in.

Reply:
Should i remove it?
Added it to tests/boards/board_shell

Reply:
lets keep the shell sample dedicated to shell features, I will need probably to move all the HW related shell to the board_shell instead.